CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## [Unreleased]
 
-- No changes yet.
+- Temporarily remove private usage check on private packages to unbreak Homebrew releases.
```

Comment:
` - Remove check for proper usage of private packages due to a breaking change made in the Golang standard library in 1.18.`